app/Services/ImportExport/AircraftImporter.php

```diff
@@ -25,9 +25,11 @@ class AircraftImporter extends ImportExport
         'subfleet'     => 'required',
         'iata'         => 'nullable',
         'icao'         => 'nullable',
+        'airport_id'   => 'required',
```

Comment:
I think this field is not mandatory at database level and some users do not check where their aircraft is, so it should be `nullable` here too for the import.